packages/api-client-core/src/operationRunners.ts

```diff
@@ -82,36 +83,44 @@ export interface ActionRunner {
     operation: string,
     defaultSelection: FieldSelection | null,
     modelApiIdentifier: string,
+    modelSelectionField: string,
     isBulkAction: true,
     variables: VariableOptions,
     options?: SelectionOptions | null,
     namespace?: string | null
-  ): Promise<Shape extends void ? void : GadgetRecordList<Shape>>;
+  ): Promise<Shape extends void ? void : GadgetRecord<Shape>[]>;
 }
 
 export const actionRunner: ActionRunner = async <Shape = any>(
   modelManager: { connection: GadgetConnection },
   operation: string,
   defaultSelection: FieldSelection | null,
   modelApiIdentifier: string,
+  modelSelectionField: string,
```

Comment:
Is this signature change gonna require a rebuild of all the built JS clients when we ship this?